PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/projects/SparkScalaSettingsStep.java

```diff
@@ -56,6 +56,9 @@ public void updateDataModel() {
 
     @Override
     public boolean validate() throws ConfigurationException {
+        if (this.builder.getScalaPluginStatus() != ScalaPluginStatus.INSTALLED) {
```

Comment:
The combined return statement is a bit complex. Make the `if (this.builder.getScalaPluginStatus() != ScalaPluginStatus.INSTALLED)` in a single line, we can emphasize that if the scala plugin not ready, we can't create the project.